### DISCUSSION TITLE
Preprint servers on PubPub?

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#1275] [General]
I'm really excited about, and intrigued by, this PubPub team post on [How to start your own preprint review community on PubPub](https://help.pubpub.org/pub/preprints-toolkit/release/4). In addition to hosting my press on PubPub, I co-lead a nonprofit preprint server for the media studies fields called [MediArXiv](https://mediarxiv.com). MediArXiv is currently hosted on the Center for Open Science's [Open Science Framework](https://mediarxiv.org), but we and some of the other independent servers have been thinking about alternatives (like PKP's emerging [Open Preprint System](https://pkp.sfu.ca/ops/)). 

Here's my question: The [PubPub post](https://help.pubpub.org/pub/preprints-toolkit/release/4) is mostly about adding a review layer on top of a preprint server hosted elsewhere. Would it be possible—and has it been tried yet—to host the server on PubPub too, together with the review layer?

**Comment by gabestein** [comment]
So glad you caught that, Jeff! Long story short, yes, we'd like to support preprint servers on PubPub in the near future. And agreed, it would work great with the review layer. It's technically very possible to do today, and we know of at least one running server in [CrimRxiv](https://www.crimrxiv.com/) that's using PubPub for it. We'd be happy to put you in touch with them to discuss their learnings.

But, as you know, our submission and review systems need to be vastly improved to really fully support it. We're [working on](https://github.com/orgs/pubpub/projects/9) some Dashboard performance improvements now, then adding some activity and notifications (which we think is a prerequisite for good submission and review), and then submission and review. So, now would actually be a great time to chat if you're considering alternatives, as we're just beginning requirements gathering for it.

All to say — if you and others are interested, reach out to me over email and let's put a call together to chat. If anyone else is interested in chatting, please ping me here and I'll reach out. We can also use this thread to share ideas/prior art/etc. in the meantime.

**Comment by jeffpooley** [comment]
Thanks for the quick reply, Gabe! We are signed up, and paid out, for 2021 with COS, so thinking about a transition, with some of these feature roll-outs possible in the calendar year, would make sense. We’ll reach out to CrimRxiv too. Really appreciate it.
